app/utils/apiUtils.js

```
@@ -2,43 +2,59 @@ import { create } from 'apisauce';
+  music: null,
   default: null
 };
 
+function resolveClientKey(type) {
+  if (type === 'music') return 'music';
+  if (type === 'github') return 'github';
+  return 'default';
+}
+
+function resolveBaseURL(key) {
+  if (key === 'music') return 'http://localhost:9000';
```

ESLint

[error] Expected { after 'if' condition.

(curly)